curvefs/src/client/s3/client_s3_cache_manager.cpp

```diff
@@ -2180,6 +2162,24 @@ void DataCache::PrepareS3ChunkInfo(uint64_t chunkId, uint64_t offset,
     return;
 }
 
+bool DataCache::CanFlush(bool force) {
+    uint64_t chunkSize = s3ClientAdaptor_->GetChunkSize();
+    uint64_t now = ::curve::common::TimeUtility::GetTimeofDaySec();
+    uint32_t flushIntervalSec = s3ClientAdaptor_->GetFlushInterval();
+
+    if (force) {
+        return true;
+    }
```

Comment:
it's better to put this condition to the begining.

Reply:
fixed